tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2913,13 +2923,23 @@ def rgb_to_yiq(images):
   Outputs a tensor of the same shape as the `images` tensor, containing the YIQ
   value of the pixels.
   The output is only well defined if the value in images are in [0,1].
+  
+  Usage Example:
+    ```
+    >>> import tensorflow as tf
+    >>> x = tf.constant([[[2.0, 5.0, 3.0]]])
+    >>> y = tf.image.rgb_to_yiq(x)
+    >>> print(y.numpy())
+    [[[ 3.875     -0.4305972 -1.6449057]]]
```

Comment:
Use fewer digits of precision? Like -0.43... -1.6... ? This will make the test more robust to different numberics